cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,50 @@
 describe('', () => {
```

Comment:
The describe function should have an appropriate description of the group of tests.